{"name": "Ошибка '404' при отправки запроса с несуществующем id на удаление курьера", "status": "passed", "steps": [{"name": "Удаляем курьера", "status": "passed", "start": 1747825198970, "stop": 1747825199147}], "start": 1747825198970, "stop": 1747825199147, "uuid": "da1c292c-6a3e-4354-a811-51d2cbf01c84", "historyId": "86fed0cfe787b4d4fc9e7470d8378f86", "testCaseId": "86fed0cfe787b4d4fc9e7470d8378f86", "fullName": "tests.test_deleted_courier.TestDeletedCourier#test_error_request_without_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_deleted_courier"}, {"name": "subSuite", "value": "TestDeletedCourier"}, {"name": "host", "value": "KOMPUTER"}, {"name": "thread", "value": "7628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_deleted_courier"}]}